app/web/src/components/Workspace/WorkspaceAuditLog.vue

```diff
@@ -190,7 +190,9 @@ const DEFAULT_FILTERS = {
   excludeSystemUser: false,
   kindFilter: [],
   serviceFilter: [],
-  changeSetFilter: [changeSetsStore.selectedChangeSetId],
+  changeSetFilter: changeSetsStore.headSelected
+    ? []
+    : [changeSetsStore.selectedChangeSetId],
```

Comment:
@stack72 What is this change?

Reply:
This removes filtering to HEAD when on HEAD - it shows all changes across the ecosystem